proto/aclk/v1/envelope.proto

```diff
@@ -0,0 +1,61 @@
+syntax = "proto3";
+
+package aclk.v1;
```

Comment:
what I wonder about are packages... we seem to be creating a mess here. We have:
```
nodeinstance::v1::UpdateNodeInstanceConnection
agent::v1::UpdateAgentConnection
nodeinstance::create::v1::CreateNodeInstance  -> already breaking pattern here
nodeinstance::create::v1::CreateNodeInstanceResult ->already breaking pattern here
```
Now we are bringing `aclk::v1::ChartDimesnionUpdate` should that be charts::v1 ? Maybe only ACLKMessagePosition should be `aclk::v1` as that might be common for other messages?
```

Reply:
Hey @underhood you 're correct, I agree it is a bit messy right now.
The thing is that initially I also followed your proposed approach but it would not compile because of circular package import. 

More specifically:

I placed the common aclk related messages in aclk.v1 so we had:`aclk.v1.ACLKMessagePosition` and `aclk.v1.ACLKMessageEnvelope`. 
The `aclk.v1.ACLKMessageEnvelope` would then include (optionally)  a collection of `chart.v1.ChartInstanceUpdate` and a collection of `chart.v1.ChartDimensionUpdate`.
But both the `chart.v1.ChartDimensionUpdate` and the `chart.v1.ChartInstanceUpdate` include the `aclk.v1.ACLKMessagePosition` as field so a circular import occurs between aclk.v1 <-> chart.v1 packages.

I'd be happy to discuss a more sensible organisation of packages and messages.

Reply:
I see lets separate that common message into `util.v1` or `aclk_lib.v1`?

Reply:
`aclk_lib.v1` sounds good to me!
Let me push another commit to introduce this and refactor what is needed.

Reply:
Just a heads up

- I extracted the common `ACLKMessagePosition` under a separate `aclk_lib.v1` package
- Imported `aclk_lib.v1` package to `ChartInstanceUpdate` and `ChartDimensionUpdate` which are now  moved out from `aclk.v1` and under `chart.v1` in separate proto files.
- The `ChartInstanceUpdate` and `ChartDimensionUpdate` messages are now imported by the `ACLKMessageEnvelope` without any issues.

Things indeed look much cleaner now IMHO (thanks Timo).

cc @underhood @stelfrag @chrikar 

PS. I also added those two missing fields in the `ChartConfig` definition.